ioctl/cmd/ws/wsprojectconfig.go

```diff
@@ -147,3 +173,28 @@ func convertCodeToZlibHex(codeFile string) (string, error) {
 
 	return hexString, err
 }
+
+// zkp vm type
+type vmType string
+
+const (
+	risc0  vmType = "risc0"  // risc0 vm
+	halo2  vmType = "halo2"  // halo2 vm
+	zkWasm vmType = "zkwasm" // zkwasm vm
+	wasm   vmType = "wasm"   // wasm vm
+)
+
+func stringToVMType(vmType string) (vmType, error) {
```

Comment:
this should be/has been done at API side

Reply:
add TODO